Twitch/Twitch.Base/Models/NewAPI/NewTwitchAPISingleDataRestResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Twitch.Base.Models.NewAPI
+{
+    /// <summary>
+    /// A wrapper result used for the New Twitch APIs where results is a single object for the data node
+    /// Used for APIs such as schedule
+    /// </summary>
+    public class NewTwitchAPISingleDataRestResult
+    {
+        /// <summary>
+        /// The data of the result.
+        /// </summary>
+        public JObject data { get; set; } 
```

Comment:
Any reason this is a JObject as opposed to a typed T class?

Reply:
Not really... just made it a bit too generic.  Was looking to bring it back as a typed class.